orttraining/orttraining/test/python/orttraining_test_orttrainer_bert_toy_onnx.py

```diff
@@ -422,28 +288,48 @@ def testToyBertCheckpointBasic():
     for k,v in loaded_sd.items():
         assert torch.all(torch.eq(v, sd[k]))
 
-
-def testToyBertCheckpointLoadZero():
+# Test ZeRO checkpoint loading from a distributed Zero stage 1 mixedprecision run to the
+# following configs. This test ensures that the weights as well as the optimizer
+# state are aggregated and loaded correctly into fp32 and fp16 single-node 
+# trainers respectively, for Adam and Lamb optimizers.
+@pytest.mark.parametrize("optimizer, mixedprecision_enabled, expected_eval_loss", [
+    (optim.LambConfig(), False, [11.011026]),
```

Comment:
where are these values coming from?

Reply:
this is the expected loss given that the checkpoint was loaded correctly